src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -388,34 +406,53 @@ async function handleListItemRequestChanges(
   if (userId === undefined) {
     throw new Error("handleListItemRequestChange Error: userId is undefined");
   }
-  const formRunnerEditUserUrl = await initialiseFormRunnerSession(list, listItem, message, isUnderTest);
+  logger.info(`user ${userId} is requesting changes for ${listItem.id}`);
+  const formRunnerEditUserUrl = await initialiseFormRunnerSession(
+    list,
+    listItem,
+    message,
+    isUnderTest
+  );
```

Comment:
I feel like there's a Prettier rule to allow a width of 120 chars. Has that been changed?